src/components/SelectDate.vue

```diff
@@ -80,17 +84,37 @@
 </template>
 
 <script>
+import { displayApi } from "../databazeFilmy";
+
 export default {
   name: "SelectDate",
   data() {
     return {
-      chosenDay: "today"
-      /*options:[
-      {text: "dnes", value: "všechna kina dnes"},
-      {text: "zítra", value: "všechna kina zítra"},
-      {text: "dalších 7 dní", value: "všechna kina dalších 7 dní"},
-      {text: "vše", value: "celý program všech kin"},]*/
+      chosenDay: "today",
+
+      apiData: fetch(
```

Comment:
Ten fetch bychom meli dat do created funkce, jak jsme to delali spolu vcera :) Kouknete tady: https://github.com/versv/dobiografu/blob/0283477e5978383586d0a1d97a9cce50bb7e5b02/src/components/cinemaProgramme.vue#L66

Takze to bude nejak tak:

```
  created() {
    fetch("https://api.apify.com/v2/datasets/mF3iLaWJ6zq5fUCSy/items?format=json&clean=1")
        .then(json => {
          console.log(json);
          return json;
        })
        .then(json => this.displayApi(json))
  },
```

plus minus... pisu aniz bych to zkousel. Pokud to nebude fungovat, tak napiste a zkusim si to u sebe.